### ISSUE TITLE
Add a linter check with github actions

### ISSUE THREAD

**Issue by Pradhvan** [scanapi/demo-api#36]
- Add a linter to the project. Can it be black?
- Add a respective GitHub action for lining analysis.

**Comment by Pradhvan** [comment]
Hi @slayer32 there is GitHub action running with the repo. 

So it would be good if you could add a GitHub action that checks linting errors when a new commit. This could also be used when a PR is made from any branch to master and on a successful merge to master.

**Comment by slayer321** [comment]
Hey @Pradhvan , thank you for adding the blog , I read it and also did my own research and I come up with this [workflow gist](https://gist.github.com/slayer321/4386f6d3dd6ed4df3e45751b037e9530)  . This is the first time I'm working with `linter` and `Github Action` so I don't know weather it  is correct or not  . Let me know what needs to be updated.

**Comment by Pradhvan** [comment]
@slayer321 yes, something similar to this. You can create a branch and add it to the [GitHub action](https://github.com/scanapi/demo-api/actions/new). Test if that works and create a PR if everything works out. 

Thank you for helping out. 😄
